pkg/kubelet/status/status_manager.go

```diff
@@ -142,7 +144,8 @@ func NewManager(kubeClient clientset.Interface, podManager kubepod.Manager, podD
 func isPodStatusByKubeletEqual(oldStatus, status *v1.PodStatus) bool {
 	oldCopy := oldStatus.DeepCopy()
 	for _, c := range status.Conditions {
-		if kubetypes.PodConditionByKubelet(c.Type) {
+		// both owned and shared conditions are used for kubelet status equality
+		if kubetypes.PodConditionByKubelet(c.Type) || kubetypes.PodConditionSharedByKubelet(c.Type) {
```

Comment:
this looks correct... if the new status has a condition the kubelet shares, and it is absent or different from the old status, the kubelet must have added or updated it

Reply:
The condition could've also been updated by an external controller (i.e. not kubelet as well) right?

Reply:
> The condition could've also been updated by an external controller (i.e. not kubelet as well) right?

correct